ProjectFinderApi/Startup.cs

```diff
@@ -122,7 +126,7 @@ public void ConfigureServices(IServiceCollection services)
 
         private static void ConfigureDbContext(IServiceCollection services)
         {
-            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
+            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? "Host=;Database=;";
```

Comment:
This looks like it's missing some values

Reply:
This is just a weird way to get around running a migration locally, I tried using profiles but it wouldn't use the connection string within it and found this line within another API, I think it doesn't care what string you use as running a migration only requires that a connection string has been set